docs/overview/architecture.md

```diff
@@ -1,10 +1,12 @@
-## LDK Architecture
-![LDK Architecture](../assets/ldk-architecture.svg)
+## Architecture
+![Architecture](../assets/architecture.png)
 
-LDK's core components are shown in the center box labeled `lightning`. The boxes
-with dotted borders are LDK's modules — these must be configured with either
-off-the-shelf or custom implementations that you provide.
+Firstly, LDK is not a daemon, but rather its own implementation of the Lightning network written as an SDK and designed to be complied and run in your own application. 
 
-`EventHandler` in the diagram is not so much a necessary LDK module, but instead
-refers to the fact that LDK generates events that the user should handle (e.g.
-the `PaymentReceived` event).
+This makes it environment agnostic, allowing you to choose your own networking stack, data storage, wallet and blockchain monitoring. In the diagram above the boxes with dotted borders are LDK's modules — these must be configured with either off-the-shelf or custom implementations that you provide.
+
+We also use an event driven archictecture that will ansychronously prompt you to do an number of things. For example LDK will generate an event when a payment is received (`PaymentReceived`). More on this in a later section. 
+
```

Comment:
Ignore if i am preempting your next step.
It could be helpful if we add a small subsection with 2-liner about each of the modules and their functionality
(Can tackle in next PR)

2-liner should probably be an answer to:
* Storage: what does LDK use this for ? 
* Broadcast transactions: what does LDK use this for ? 
etc.

Need not be too detailed in overview section.

Reply:
Yeah the plan is to provide examples for both of these in the following PR's. 
I plan to tackle Peer management, Channel management, Payments & Routing and their dependencies.

Reply:
That was going to be my feedback after seeing the preview...I liked the organization of Peer management, Channel management, and Payments & Routing from the btc++ prezo.